sdk/communication/Azure.Communication.CallAutomation/src/CallAutomationClient.cs

```diff
@@ -633,6 +645,12 @@ private CreateCallRequestInternal CreateCallRequest(CreateGroupCallOptions optio
                 Source = Source == null ? null : new CommunicationUserIdentifierModel(Source.Id),
             };
 
+            // Add custom cognitive service domain name
+            if (options.AzureCognitiveServicesEndpointUri != null)
+            {
+                request.AzureCognitiveServicesEndpointUrl = options.AzureCognitiveServicesEndpointUri.AbsoluteUri;
```

Comment:
Will need this in create group call as well. I don't see those changes can you please confirm?

Reply:
discussed offline